.github/workflows/pushrelease.yml

```diff
@@ -332,13 +332,37 @@ jobs:
           install.packages("pkgdown", type = "binary")
         shell: Rscript {0}
 
+       # install packages needed
+      - name: Install rmarkdown
+        run: Rscript -e 'install.packages("rmarkdown")'
+
+      # Install ganttrify
+      - name: Install ganttrify
+        run: Rscript -e 'remotes::install_github("giocomai/ganttrify")'
```

Comment:
Why does this need to be its own step and not just included among the other non_CRAN packages?